lib/actions/device.coffee

```diff
@@ -432,3 +512,51 @@ exports.init =
 				return device.uuid
 
 		.nodeify(done)
+
+exports.getStatus =
+	signature: 'device status <device>'
+	description: 'get the status of a device'
+	help: '''
+		Use this command to get the status of a device
+
+		Examples:
+
+			$ resin device status 601317
+	'''
+	permission: 'user'
+	action: (params, options, done) ->
+		resin = require('resin-sdk-preconfigured')
+		resin.models.device.getStatus(params.device).then (status) ->
+			console.log(status)
```

Comment:
This is wrong - this was me looking at the parameter (which _should_ be an object), not the response - that is indeed fine to log to the console if that's what you're looking for.